routes/HomePage.vue

```diff
@@ -128,7 +128,11 @@ export default {
       account: accountId.value,
       preventRenotify: true
     }));
-
+    //refresh has to call done
+    const refresh = async done => {
+      await getCredentials();
+      done();
+    };
```

Comment:
We should handle the case that something goes wrong with `getCredentials()`:

```suggestion
    // refresh has to call `done` to complete
    const refresh = async done => {
      try {
        await getCredentials();
      } finally {
        done();
      }
    };
```

Reply:
this might be redundant as `getCredentials` is already inside of a try, catch, finally block.
The redundancy though might be desirable as changes could be made to `getCredentials` in the future that might make it throw. LMK if you still want the finally taken the function being called is already inside of a try, catch, & finally block.

https://github.com/digitalbazaar/bedrock-vue-wallet/blob/ea2ae93e4eb024b26216f0ef6503067a188e8254/routes/HomePage.vue#L66-L102